sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-data.js

```diff
@@ -256,19 +257,33 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
                 output.appendChild(cvs);
             })();
         }
+        // XXX: remove this stuff
+        //if (bindingTarget == gl.TEXTURE_2D_ARRAY) {
+        //    return;
+        //}
+        //if (true) { const sub = true;
+        //    if (true) { const flipY = false;
+        //        if (true) { const premul = false;
+        //            if (true) { var rect = rects[1];
+        //                if (true) { const rTexCoord = 0;
         for (const sub of [false, true]) {
             for (const flipY of [false, true]) {
                 for (const premul of [false, true]) {
-                    for (var irect = 0; irect < rects.length; irect++) {
+                    for (let irect = 0; irect < rects.length; irect++) {
```

Comment:
Based on MDN (plus a quick test in Safari), yes, let and const are well supported.